app/main.py

```diff
@@ -1,3 +1,59 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float) -> None:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+
+        return Distance(self.km + other)
+
+    def __iadd__(self, other: int | float) -> None:
+        if isinstance(other, Distance):
+            self.km += other.km
+            return self
+        elif isinstance(other, (int | float)):
+            self.km += other
+            return self
+
+    def __mul__(self, other: int | float) -> None:
+        return Distance(self.km * other)
+
+    def __truediv__(self, other: int | float) -> None:
+        return Distance(round(self.km / other, 2))
+
+    def __lt__(self, other: int | float) -> bool:
+        if isinstance(other, Distance):
+            return self.km < other.km
+        elif isinstance(other, (int | float)):
+            return self.km < other
+
+    def __gt__(self, other: int | float) -> bool:
+        if isinstance(other, Distance):
+            return self.km > other.km
+        elif isinstance(other, (int | float)):
+            return self.km > other
+
+    def __eq__(self, other: int | float) -> bool:
+        if isinstance(other, Distance):
+            return self.km == other.km
+        elif isinstance(other, (int | float)):
+            return self.km == other
+
+    def __le__(self, other: int | float) -> bool:
+        if isinstance(other, Distance):
+            return self.km <= other.km
+        elif isinstance(other, (int | float)):
```

Comment:
```suggestion
        if isinstance(other, (int | float)):
```
use just if, because of the return statement in the 51st line. please, fix everywhere